samcli/lib/init/templates/cookiecutter-aws-sam-hello-nodejs/cookiecutter.json

```diff
@@ -1,4 +1,4 @@
 {
     "project_name": "Name of the project",
-    "runtime": "nodejs12.x"
+    "runtime": "nodejs14.x"
```

Comment:
Same as the other one, should we keep original version inside SAM CLI?

Reply:
So with the current INIT design we default to the latest version of the runtime selected by the customer for hello world templates however customers can still specify the runtime version they would like to use. So defaulting to node14.x as against nodejs12.x was necessary and after working on this project I don't see how this affects backward compatibility.